src/lib/plugin.js

```diff
@@ -236,6 +236,7 @@ class FiveBellsLedger extends EventEmitter2 {
       this.connection = reconnect({immediate: true}, (ws) => {
         ws.on('open', () => {
           this.connected = true
+          this.emit('connect')
```

Comment:
Doesn't putting `emit('connect')` here mean that if the connection fails and retries, `'connect'` will be emitted a second time, without `'disconnect'` ever being emitted? This seems like unexpected behavior.